Classes/Views/DotListView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  DotView.swift
+//  Freetime
+//
+//  Created by Joe Rocca on 10/19/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class DotListView: UIView, UICollectionViewDataSource {
+    
+    private let collectionView: UICollectionView
+    private var colors = [UIColor]()
+    
+    init(frame: CGRect = .zero,
+         dotSize: CGSize = CGSize(width: 10, height: 10)) {
+        
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        layout.itemSize = dotSize
+        layout.minimumLineSpacing = Styles.Sizes.columnSpacing / 2.0
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.backgroundColor = .clear
+        collectionView.isUserInteractionEnabled = false
+        collectionView.register(IssueLabelDotCell.self, forCellWithReuseIdentifier: IssueLabelDotCell.reuse)
+        
+        super.init(frame: frame)
+        
+        collectionView.dataSource = self
+        addSubview(collectionView)
+        collectionView.snp.makeConstraints { (make) in
+            make.edges.equalTo(self)
+        }
```

Comment:
Could we move this to `layoutSubviews` and just do `collectionView.frame = bounds` instead? Cheaper, simpler.

Reply:
Done!